Gemfile

```diff
@@ -44,6 +44,7 @@ group :development, :test do
   gem 'vcr',                        '~> 3.0'
   gem 'shoulda-matchers',           '~> 3.1'
   gem 'rake',                       '~> 10.4'
+  gem 'selenium-webdriver'
```

Comment:
We have poltergeist already in the gem file, could we use that instead of selenium please 😄